src/routes/event/list.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  Box,
+  Button,
+  Container,
+  Drawer,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Menu,
+  MenuItem,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { auth, database } from "../../firebase";
+import { get, ref } from "firebase/database";
+import { useState } from "react";
+import { useCounterState } from "../../states/useCounterState";
+import { onAuthStateChanged } from "firebase/auth";
+import { useNavigate } from "react-router-dom";
+import * as React from "react";
+
+export default function EventList() {
+  //const { counter, fetchCounter } = useCounterState();
+  const [firstName, setFirstName] = useState("");
+  const navigate = useNavigate();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  onAuthStateChanged(auth, (user) => {
+    if (user) {
+      get(ref(database, "users/" + auth.currentUser?.uid + "/firstName"))
+        .then((snapshot) => {
+          if (snapshot.exists()) {
+            setFirstName(snapshot.val());
+            console.log(firstName);
+          } else {
+            console.log("No data available");
+          }
+        })
+        .catch((error) => {
+          console.error(error);
+        });
+    } else {
+      console.log("No user signed in");
+      navigate("/");
+    }
+  }); // marginLeft 150 somehow too much
```

Comment:
same as the above comment, 
this also needs to be put in the context, but i guess it will be in the next PR.